specification/search/data-plane/Azure.Search/preview/2023-07-01-Preview/searchservice.json

```diff
@@ -0,0 +1,11599 @@
+          "description": "The execution limits for the indexer."
+        }
+      },
+      "required": [
+        "status",
+        "executionHistory",
+        "limits"
+      ],
+      "description": "Represents the current status and execution history of an indexer."
+    },
+    "IndexerStatus": {
+      "type": "string",
+      "enum": [
+        "unknown",
+        "error",
+        "running"
+      ],
+      "x-ms-enum": {
+        "name": "IndexerStatus",
+        "modelAsString": false,
+        "values": [
+          {
+            "value": "unknown",
+            "name": "Unknown",
+            "description": "Indicates that the indexer is in an unknown state."
+          },
+          {
+            "value": "error",
+            "name": "Error",
+            "description": "Indicates that the indexer experienced an error that cannot be corrected without human intervention."
+          },
+          {
+            "value": "running",
+            "name": "Running",
+            "description": "Indicates that the indexer is running normally."
+          }
+        ]
+      },
+      "x-nullable": false,
+      "description": "Represents the overall indexer status."
+    },
+    "IndexerExecutionStatusDetail": {
+      "type": "string",
+      "enum": [
+        "resetDocs"
+      ],
+      "x-ms-enum": {
+        "name": "IndexerExecutionStatusDetail",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "resetDocs",
+            "name": "ResetDocs",
+            "description": "Indicates that the reset that occurred was for a call to ResetDocs."
+          }
+        ]
+      },
+      "description": "Details the status of an individual indexer execution."
+    },
+    "IndexerCurrentState": {
+      "properties": {
+        "mode": {
+          "$ref": "#/definitions/IndexingMode",
+          "readOnly": true,
+          "description": "The mode the indexer is running in."
+        },
+        "allDocsInitialChangeTrackingState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Change tracking state used when indexing starts on all documents in the datasource."
+        },
+        "allDocsFinalChangeTrackingState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Change tracking state value when indexing finishes on all documents in the datasource."
+        },
+        "resetDocsInitialChangeTrackingState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Change tracking state used when indexing starts on select, reset documents in the datasource."
+        },
+        "resetDocsFinalChangeTrackingState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Change tracking state value when indexing finishes on select, reset documents in the datasource."
+        },
+        "resetDocumentKeys": {
+          "type": "array",
+          "items":
```

Comment:
Nit - I don't think this will render in the docs.  It might be nice to also update the description to say it's deprecated and point users at a replacement.